catalogue/webapp/components/ImageEndpointSearchResults/ImageEndpointSearchResults.tsx

```diff
@@ -94,47 +82,71 @@ const ImageEndpointSearchResults: FunctionComponent<Props> = ({
     [images]
   );
 
-  const imageRenderer = useCallback(galleryImage => {
-    const photo: GalleryImageProps = galleryImage.photo;
+  const AlbumRow = styled.ul`
+    display: flex;
+    flex-flow: row no-wrap;
+    align-items: space-between;
+    margin-bottom: 0;
+  `;
+
+  const renderRowContainer: RenderRowContainer = ({ children }) => {
+    return <AlbumRow>{children}</AlbumRow>;
+  };
+
+  const ImageFrame = styled.div`
+    display: block;
+    position: relative;
+    width: 100%;
+    height: 100%;
+  `;
+
+  const imageRenderer: React.FunctionComponent<
+    RenderPhotoProps<GalleryImageProps>
+  > = ({ photo, layout, wrapperStyle }) => {
+    // these are values and props that are passed in by the PhotoAlbum component
     const rgbColor = hexToRgb(photo.averageColor || '');
-
     return (
-      <ImageContainer key={galleryImage.key} role="listitem">
-        <ImageCard
-          id={photo.id}
-          workId={photo.source.id}
-          image={{
-            contentUrl: photo.src,
-            width: photo.width - imageMargin * 2,
-            height: photo.height,
-            alt: photo.source.title,
-          }}
-          onClick={event => {
-            event.preventDefault();
-            setExpandedImage(photo);
-            setIsActive(true);
-          }}
-          layout="fixed"
-          background={
-            background ||
-            (rgbColor &&
-              `rgba(${rgbColor.r}, ${rgbColor.g}, ${rgbColor.b}, 0.5)`)
-          }
-        />
-      </ImageContainer>
+      <li role="listitem" style={wrapperStyle}>
+        <ImageFrame>
+          <ImageCard
+            id={photo.id}
+            workId={photo.source.id}
+            image={{
+              contentUrl: photo.src,
+              width: layout.width,
+              height: layout.height,
+              alt: photo.source.title,
+            }}
+            onClick={event => {
+              event.preventDefault();
+              setExpandedImage(photo);
+              setIsActive(true);
+            }}
+            layout="fixed"
+            background={
+              background ||
+              (rgbColor &&
+                `rgba(${rgbColor.r}, ${rgbColor.g}, ${rgbColor.b}, 0.5)`)
```

Comment:
nit: if this is the only place `rgbColor` is used, I'd be tempted to move this logic outside this bit of templating